doc/README.md

```diff
@@ -360,19 +361,30 @@ while (!idList.done) {
 **Register an Identity in a Realm**
 
 ```js
+// Note: Identities should be registered using Tozny's JS SDK (@toznysecure/sdk/node) instead of the account client. The account client's registerIdentity() is intended for internal use.
+
 // Create a token
 const token = await accountClient.newRegistrationToken(tokenName, permissions)
-const identity = {
-  name: 'identityName',
-  email: 'identity@example.com',
-  first_name: 'firstName',
-  last_name: 'lastName',
-}
-// Register Identity
-const registeredIdentity = await accountClient.registerIdentity(
+
+// Create a Realm object
+const realmName = 'realmName'
+const appName = 'account'
+const brokerTargetUrl = 'http://platform.local.tozny.com:8000'
```

Comment:
this isn't the right format for this, see https://github.com/tozny/flutter_plugin#end-to-end-tozid-identity-registration-login--tozstore-encrypted-filed-storage-example for an example of what this looks like.